db/seeds.rb

```diff
@@ -1,13 +1,4 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-
-
-john = User.create(username: 'user',password: 'password', first_name: 'John', last_name: 'Doe', email: 'example@example.com', role: 1)
+john = User.create(username: 'user',password: 'password', first_name: 'John', last_name: 'Doe', display_name: "MikeDaoSellsCelluliteProducts", email: 'example@example.com', role: 1)
```

Comment:
Useless assignment to variable - `john`.<br>Line is too long. [181/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.